tools/otci/openthread_otci.egg-info/PKG-INFO

```diff
@@ -0,0 +1,73 @@
+Metadata-Version: 2.4
+Name: openthread-otci
+Version: 0.0.1
+Summary: OpenThread Control Interface
+Home-page: https://github.com/openthread/openthread
+Author: The OpenThread Authors
+Author-email: The OpenThread Authors <openthread-users@googlegroups.com>
+Project-URL: Homepage, https://github.com/openthread/openthread
+Project-URL: Issues, https://github.com/openthread/openthread/issues
+Classifier: Programming Language :: Python :: 3
+Classifier: License :: OSI Approved :: BSD License
+Classifier: Operating System :: OS Independent
+Requires-Python: >=3.6
+Description-Content-Type: text/markdown
+License-File: LICENSE
+Dynamic: author
+Dynamic: home-page
+Dynamic: license-file
+Dynamic: requires-python
+
+# OpenThread Control Interface
+
+The OpenThread Control Interface (OTCI) is a library which provides uniform python interfaces to connect and control various kinds of devices running OpenThread.
```

Comment:
```suggestion
The OpenThread Control Interface (OTCI) is a library which provides uniform Python interfaces to connect and control various kinds of devices running OpenThread.
```

Reply:
will update this in the README.md (the PKG-INFO shouldn't have been part of the pr, so will remove that)